src/components/cart/CartOrderSummary.test.tsx

```diff
@@ -9,18 +9,22 @@ import {
   CartsFactory,
 } from 'boclips-api-client/dist/test-support/CartsFactory';
 import { CartValidationProvider } from 'src/components/common/providers/CartValidationProvider';
+import { UserFactory } from 'boclips-api-client/dist/test-support/UserFactory';
 import { BoclipsClientProvider } from '../common/providers/BoclipsClientProvider';
 
 describe('Cart Order Summary', () => {
-  it('displays copy describing pricing for additional services if selected', async () => {
+  it('displays new copy describing pricing for additional services if selected and user has feature flag', async () => {
```

Comment:
Testing this logic could be done in a component test and here we would test what users sees as BO_WEB_APP_DEV is for internal users